pom.xml

```diff
@@ -64,6 +64,7 @@
     <maven.wagon.http.retryHandler.class>standard</maven.wagon.http.retryHandler.class>
     <!-- Try at most 6 times to deploy every artifact -->
     <maven.wagon.http.retryHandler.count>6</maven.wagon.http.retryHandler.count>
+    <project.build.outputTimestamp>2020-05-17T18:04:00Z</project.build.outputTimestamp>

```

Comment:
AIUI, one of the goals of using the theoryinpractice plugin was to have the timestamp autogenerated.  

Before that, we had a variable so the RM could choose a timestamp helps identify the release (using 9:07 for 0.9.7).

What are the advantages of explicitly setting the timestamp in this change?

Reply:
> What are the advantages of explicitly setting the timestamp in this change?

it is also auto-generated (by maven-release-plugin), basically stored in pom.xml
the advantage is just to remove one additional plugin that does runtime magic with just a simple property updated during release
theoryinpractice just did a workaround early, before maven-release-plugin 3.0.0-M1 was ready

Reply:
Just to make sure I understand:  The maven-release-plugin will overwrite the timestamp in project.build.outputTimestamp, so we can't really control the timestamp manually.  That's ok, I guess, just want to make sure our release instructions don't tell the RM to set it.

Reply:
yes, that's it: it will update to match the timestamp of the release process, without requiring any RM action